kolibri/plugins/user/assets/src/actions.js

```diff
@@ -22,6 +23,60 @@ function showRoot(store) {
   });
 }
 
+function editProfile(store, edits, session) {
```

Comment:
`session` doesn't need to be a parameter here - it can be retrieved from the store directly:

```javascript
const session = store.state.core.session;
```